README.md

```diff
@@ -14,7 +14,7 @@ Read more in the [documentation on ReadTheDocs](https://<RTD_NAME>.readthedocs.i
 ## Quickstart
 
 ```sh
-pip install <PYPI_NAME>
+python -m pip install <PYPI_NAME>
```

Comment:
what's the thinking behind this inclusion?

Reply:
It's recommended: 
https://snarky.ca/why-you-should-use-python-m-pip/
https://packaging.python.org/en/latest/tutorials/installing-packages/

TL;DR, it ensures that you're installing to the same instance of python that you use for running.